src/components/drag-and-drop/drag-and-drop.stories.js

```diff
@@ -54,21 +59,21 @@ export const Default = () => {
   return (
     <DraggableContext autoScroll={autoScroll} onDrag={handleDrag}>
       <div>
-        <Table tbody={false}>
+        <FlatTable tbody={false}>
```

Comment:
`tbody` prop does not exist in `FlatTable` (remove this from other files as well)